### DISCUSSION TITLE
fclones-gui 0.1.2

### DISCUSSION THREAD

**Discussion by pkolaczk** [pkolaczk/fclones#215] [Announcements]
## What's Changed
* Fix file stats not cleared before another dupes scan by @pkolaczk in https://github.com/pkolaczk/fclones/pull/214
* Update fclones core to 0.32.1

**Full Changelog**: https://github.com/pkolaczk/fclones/compare/gui-v0.1.1...gui-v0.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/pkolaczk/fclones/releases/tag/gui-v0.1.2'>fclones-gui 0.1.2</a>.</em>

**Comment by yehia2amer** [comment]
That's is Awesome @pkolaczk !!!
